tests/test_time_of_day.py

```
@@ -14,166 +14,164 @@
-        self.assertIs(type(sec_from_hms(0, 0, 0., 0)), float)
-        self.assertIs(type(sec_from_hms(0, 0., 0, 0)), float)
-        self.assertIs(type(sec_from_hms(0., 0, 0, 0)), float)
-
-        microsec = np.arange(0, 1000000, 10).reshape(25, 4, -1)
-        sec = sec_from_hms(0, 0, 0, microsec)
-        self.assertTrue(np.all(microsec / 1.e6 == sec))
-        self.assertEqual(sec.shape, microsec.shape)
-
-        self.assertRaises(JVF, sec_from_hms, 0,  0,  0, -1, validate=True)
-        self.assertRaises(JVF, sec_from_hms, 0,  0,  0, 1000000, validate=True)
+def test_time_of_day():
```

Ruff (0.15.6)

[warning] Too many statements (133 > 50)

(PLR0915)